src/screens/LoginPage/LoginPage.module.css

```diff
@@ -111,6 +111,11 @@ form > input {
   cursor: pointer;
   transition: transform 0.2s, box-shadow 0.2s;
   width: 100%;
+}
+.whiteloginbtn:hover{
+background-color:#737373;
+color:white;
```

Comment:
some consistency would be nice so either use the color name or the hex code ( i prefer the hex code )